protocol/x/clob/abci.go

```diff
@@ -198,7 +200,27 @@ func PrepareCheckState(
 
 	// 6. Get all potentially liquidatable subaccount IDs and attempt to liquidate them.
 	subaccountIds := liquidatableSubaccountIds.GetSubaccountIds()
-	if err := keeper.LiquidateSubaccountsAgainstOrderbook(ctx, subaccountIds); err != nil {
+	numDeleveragingAttempts, err := keeper.LiquidateSubaccountsAgainstOrderbook(ctx, subaccountIds)
+	if err != nil {
+		panic(err)
+	}
+
+	// 7. Deleverage subaccounts with open positions in final settlement markets.
```

Comment:
organizational suggestion: 

instead of having step 6 being liquidate + -ve TNC deleveraging, let's make step 6 liquidation only and return []subaccounts to deleverage instead of `numDeleveragingAttempts`

step 7 can be deleveraging for both -v TNC and final settlement.  wdyt?

Reply:
actually this might be a little weird because of how the deleveraging attempts from step 6 come from a list of unfilled liquidation orders. Will think about this and make the change if I think it makes sense. I see your point

Reply:
decided to follow your advice, updating now